HACKING.md

```diff
@@ -76,7 +76,7 @@ To update the blackbox compiler tests set TeamCity build number in `gradle.prope
 
 * **-Pprefix** allows one to choose external test directories to run. Only tests from directories with given prefix will be executed.
 
-        ./gradlew -Pprefix=external_codegen_box_cast run_external
+        ./gradlew -Pprefix=build_compiler_external_codegen_box_cast run_external
```

Comment:
`build_external_compiler_codegen_box_cast`?

Reply:
My bad. Amended the commit.

Reply:
Thanks!